Doc/reference/datamodel.rst

```diff
@@ -1437,6 +1448,9 @@ Basic customization
 
    The return value must be a string object.
 
+   The default implementation by the ``object`` class should be given
```

Comment:
```suggestion
   The default implementation by the :class:`object` class should be given
```

Resolve the reference since we're not actually inside `object` proper right now, and to get consistent formatting and semantics.